.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  test:
+    name: Build and Publish
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: [2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7]
```

Comment:
Do we still need to support these really old ruby versions?  In fact, all of these ruby versions are EOL: https://www.ruby-lang.org/en/downloads/branches/

Reply:
We don't have a version support policy of any kind, and our Tollbelt itself uses Ruby 2.3, so I'm leaving it as is since they still work and pass on GHA.
(It would also balloon the scope of migrating these 50+ repos off of Travis if I had to eliminate tech debt in each of them)